.gitignore

```diff
@@ -26,3 +26,6 @@ massif-*
 
 # CMake generates *.dir/ folders for in-tree builds (used by MSVC projects), ignore all of those:
 *.dir/
+
```

Comment:
I'm sorry, could you please give the rationale for this change? It doesn't seem to be related to the issue this PR sets out to solve.

Reply:
Yes, not related to this issue, but just for vscode user.
Will revert it.

Reply:
fixed